pkg/cmd/server/origin/master.go

```diff
@@ -1063,6 +1067,51 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	controller.Run()
 }
 
+func (c *MasterConfig) ensureDefaultSecurityContextConstraints() {
+	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
+	if err != nil {
+		glog.Fatalf("Unable to initialize security context constraints: %v", err)
+	}
+	if len(sccList.Items) > 0 {
+		return
+	}
+	glog.Infof("No security context constraints detected, adding defaults")
+	sccAdmin := &kapi.SecurityContextConstraints{
+		ObjectMeta: kapi.ObjectMeta{
+			Name: "admin",
+		},
+		AllowPrivilegedContainer: true,
+		AllowHostDirVolumePlugin: true,
+		SELinuxContext: kapi.SELinuxContextStrategyOptions{
+			Type: kapi.SELinuxStrategyRunAsAny,
+		},
+		RunAsUser: kapi.RunAsUserStrategyOptions{
+			Type: kapi.RunAsUserStrategyRunAsAny,
+		},
+		Groups: []string{"system:authenticated"},
+	}
+	_, err = c.KubeClient().SecurityContextConstraints().Create(sccAdmin)
+	if err != nil {
+		glog.Fatalf("Unable to create default security context constraint %#v.  Got error: %v", sccAdmin, err)
+	}
+
+	sccUser := &kapi.SecurityContextConstraints{
+		ObjectMeta: kapi.ObjectMeta{
+			Name: "user",
+		},
+		SELinuxContext: kapi.SELinuxContextStrategyOptions{
+			Type: kapi.SELinuxStrategyMustRunAs,
+		},
+		RunAsUser: kapi.RunAsUserStrategyOptions{
+			Type: kapi.RunAsUserStrategyMustRunAsRange,
```

Comment:
Yes for the SELinux strategy since it is an exact match, no for the user strategy.  The most restrictive user strategy would be `MustRunAs` with a single UID.